cpu/sam0_common/Makefile.dep

```diff
@@ -2,6 +2,10 @@ ifneq (,$(filter periph_uart_nonblocking,$(USEMODULE)))
   USEMODULE += tsrb
 endif
 
+ifneq (,$(filter periph_rtc periph_rtt,$(USEMODULE)))
+  FEATURES_REQUIRED += periph_rtc_rtt
```

Comment:
IMO having the RTC and RTT implementation merged is not a new feature, but rather an implementation choice. So IMO no `periph_rtc_rtt` feature should be introduced, but rather let `periph_rtc` and `periph_rtt` depend on the *module* (not *feature*) `periph_rtc_rtt`.

Have you considered to just drop the `periph_rtc` implementation and use `drivers/rtt_rtc` instead?

Reply:
> So IMO no `periph_rtc_rtt` feature should be introduced, but rather let `periph_rtc` and `periph_rtt` depend on the _module_ (not _feature_) `periph_rtc_rtt`.

Alright!

> Have you considered to just drop the `periph_rtc` implementation and use `drivers/rtt_rtc` instead?

No - if we have a RTC hardware implementation we should not use the software emulation. It would add unnecessary overhead and wake up the CPU periodically on overflow.